apps/src/levelbuilder/ai-tutor/types.ts

```diff
@@ -0,0 +1,9 @@
+import {ValueOf} from '@cdo/apps/types/utils';
+import {AiChatModelIds} from '@cdo/generated-scripts/sharedConstants';
+
+import {AI_TUTOR_ENDPOINT, LLM_GUARD_ENDPOINT} from './constants';
+
+export type Endpoint =
```

Comment:
@Erin007 no functional changes to the AI Tutor/Gen AI tester, just adding some types since we have the `AiChatModelIds` type now